js/core/src/reflection.ts

```diff
@@ -226,9 +226,13 @@ export class ReflectionServer {
 
     server.post('/api/notify', async (request, response) => {
       const { telemetryServerUrl, reflectionApiSpecVersion } = request.body;
-      if (typeof telemetryServerUrl === 'string') {
-        setTelemetryServerUrl(telemetryServerUrl);
-        logger.debug(`Connected to telemetry server on ${telemetryServerUrl}`);
+      if (!process.env.GENKIT_TELEMETRY_SERVER) {
+        if (typeof telemetryServerUrl === 'string') {
```

Comment:
Why this check?

Reply:
I was preserving the original check. I reckon it is a type check